README.md

```diff
@@ -367,8 +371,8 @@ spec:
   name: "shared-example-node-workload"
   allCores: true
   reservedCPUs:
-    - 0
-    - 1
+    - cores: [0, 1]
```

Comment:
Would "cpus" be more accurate?

Reply:
I think so - will need to think a bit more but I will add as an improvement in the upcoming release.

Reply:
Doesn't this need to be fixed now for the 2.4.0 release? Otherwise you will be introducing another backwards incompatible change in the next release.